1st iteration

Eugetis opened the pull request.

Eugetis committed:
Merge pull request #29 from Eugetis/modal/card-modal

feat: event modal layout

dmitry-razumov committed:
feat: tag-filters

dmitry-razumov committed:
Merge branch 'develop' into feat/tag-filters

GeorgyMedvedsky committed:
update to scss mixins

Eugetis committed:
Merge pull request #31 from Eugetis/feat/fields

fix: fields update to scss mixins

Eugetis committed:
Merge pull request #30 from Eugetis/feat/tag-filters

ui-kit: tag filters

dmitry-razumov committed:
fix: ui-kit div<>button для фильтров-тэгов

Eugetis committed:
Merge pull request #32 from Eugetis/feat/tag-filters

fix: ui-kit div<>button для фильтров-тэгов

dmitry-razumov committed:
feat: controllers: floating buttons

Eugetis committed:
Merge pull request #33 from Eugetis/feat/controllers

ui-kit: floating buttons

dmitry-razumov committed:
feat: amount controller

dmitry-razumov committed:
feat: map pointer & map zoom

Eugetis committed:
Merge pull request #34 from Eugetis/feat/controllers

ui-kit: controllers

marahin1992 committed:
created additional pages

Commit:
fix styles

Commit:
update

Eugetis committed:
Merge pull request #35 from Eugetis/feat/page/additional-pages

created additional pages

dmitry-razumov committed:
feat: tab switcher

Eugetis committed:
Merge pull request #36 from Eugetis/feat/participants

fix: participants page layout

Eugetis committed:
Merge pull request #37 from Eugetis/feat/tab-switch

ui-kit: tab switcher layout

Eugetis committed:
refact: huge layout refactoring

Eugetis committed:
fix: resolved old conflict

Eugetis committed:
test

Eugetis committed:
Merge branch 'refact/layout' into develop

Eugetis committed:
fix: resolved conflicts after layout refact

dmitry-razumov committed:
refact: tab-switcher
refact: tab-switcher вставил в catalog.html
feat: аккуратно добавил пока в index.js реализацию работы переключателя и контента на странице
внимание! временно заремарил секцию когда нет карточек
по уму для неё надо завести свой класс и вкл\выкл селектор типа
catalog__no-events-container catalog__no-events-container_opened
тогда я смогу разрулить подусловием в js в переключателе
что показывать - "карточки" или "нет мероприятий"

Eugetis committed:
Merge pull request #39 from Eugetis/feat/tab-switch

refact: tab-switcher

Eugetis committed:
refact: layout next

Eugetis committed:
Merge pull request #40 from Eugetis/refact/layout-new

refact: layout next

Eugetis committed:
feat/js-prepared

Eugetis committed:
Merge pull request #41 from Eugetis/feat/prepare-js

feat: prepared js structure

Commit:
Добавил 2 функции, getCards, getCities -> для получения данных с API

Eugetis committed:
Merge pull request #42 from Eugetis/api/json

Добавил 2 функции, getCards, getCities -> для получения данных с API

GeorgyMedvedsky committed:
fix: margin

Commit:
Добавил отрисовку карточек

Commit:
Добавил добавление/удаления лайков, localstorage

GeorgyMedvedsky committed:
Merge branch 'develop' into feat/participants

AlexxxDevv committed:
дропдаун

AlexxxDevv committed:
Merge branch 'develop' of github.com:Eugetis/hedonism-fest into develop

Commit:
Добавил открывания модалки при клике на карточку в грид секции

Commit:
Маленькая прафа в api.js

AlexxxDevv committed:
Revert "Merge branch 'develop' of github.com:Eugetis/hedonism-fest into develop"

This reverts commit 649c538fa7b43a70df2c6f5b385f2f63820d5922, reversing
changes made to 52b0cf1879adfb72c042bf94a2f5366a3678ce4c.

AlexxxDevv committed:
dropdown

GeorgyMedvedsky committed:
feat: create fieldset switching

GeorgyMedvedsky committed:
Merge branch 'develop' into feat/participants

AlexxxDevv committed:
добавлен localstorage

AlexxxDevv committed:
отменил удаление api.js и catalog.js

AlexxxDevv committed:
Merge branch 'develop' of github.com:Eugetis/hedonism-fest into develop

Eugetis committed:
Merge branch 'develop' into catalog/card-render

Eugetis committed:
Merge pull request #43 from Eugetis/catalog/card-render

Добавил отрисовку карточек

Eugetis committed:
Merge branch 'develop' into event/local-storage

Eugetis committed:
Merge pull request #44 from Eugetis/event/local-storage

Event/local storage

Eugetis committed:
Merge branch 'develop' into modal/card-template

Eugetis committed:
Merge pull request #46 from Eugetis/modal/card-template

Modal/card template

AlexxxDevv committed:
Merge pull request #47 from Eugetis/develop

Develop

Commit:
Исправлена ошибка с some

Eugetis committed:
Merge pull request #48 from Eugetis/modal/card-template

Исправлена ошибка с some

Commit:
Мелкий фикс того же some

Eugetis committed:
Merge pull request #49 from Eugetis/modal/card-template

Мелкий фикс того же some

Commit:
Добавил обрезание по 3 символа в месяце

GeorgyMedvedsky committed:
fix: update pr

AlexxxDevv committed:
добавлено сохранение галочки выбора

AlexxxDevv committed:
белая цвет галки для инпута

Commit:
Пофиксил модальное окно, написал комментарии ко всем функциям

Commit:
table-lines__value - text-align: end -> добавлен

Eugetis committed:
Merge pull request #51 from Eugetis/modal/card-template

Добавил обрезание по 3 символа в месяце

Eugetis committed:
Merge pull request #52 from Eugetis/feat/participants

feat: js switching form steps

Eugetis committed:
Merge pull request #53 from Eugetis/header-with-js

Header with js

AlexxxDevv committed:
сьезжающий хедер при скролле

Eugetis committed:
fix: some js errors

Eugetis committed:
Merge pull request #54 from Eugetis/header-with-js

feat: scroll header

Eugetis committed:
fix: some fixes

dmitry-razumov committed:
feat: catalog js переключатель, загрузка\отображение карты, смена города на карте

dmitry-razumov committed:
merge: with actual develop

marahin1992 committed:
new file photo-slider.js

Eugetis committed:
Merge pull request #55 from Eugetis/feat/photo-slider

new file photo-slider.js

marahin1992 committed:
footer-stile_modificator

dmitry-razumov committed:
refact: обновление карты по выбранному городу

dmitry-razumov committed:
Merge branch 'develop' into catalog/map

Eugetis committed:
Merge pull request #56 from Eugetis/feat/footer-modificator

feat: footer-stile modificator, js

dmitry-razumov committed:
refact: почистил код, добавил комментариев

Commit:
Добавил лайк в localStorage и локальное отображение в реальном режиме у кнопки модального окна - хочу пойти, кнопка полностью интерактивная и связана с карточками на странице

Eugetis committed:
Merge pull request #57 from Eugetis/catalog/map

Catalog: map

Commit:
Добавил комменты к функциям

Eugetis committed:
Merge pull request #58 from Eugetis/modal/card-template

feat: js like card

Eugetis committed:
fix: card layout

Eugetis committed:
Merge branch 'develop' into fix/card-layout

Eugetis committed:
Merge pull request #59 from Eugetis/fix/card-layout

fix: card layout

Commit:
Добавил реальные карточки на главную страницу, исправил ошибки в консоли

Commit:
Добавил display:none

Eugetis committed:
Merge pull request #60 from Eugetis/modal/card-template

Добавил реальные карточки на главную страницу, исправил ошибки в консоли

AlexxxDevv committed:
стили дропдауна, дефолтный город в storage

AlexxxDevv committed:
приподнят на 16px дропдаун

Eugetis committed:
Merge pull request #61 from Eugetis/header-with-js

стили дропдауна, дефолтный город в storage

AlexxxDevv committed:
убран лишний маржин, решена проблема с переносом

Eugetis committed:
Merge pull request #62 from Eugetis/header-with-js

убран лишний маржин, решена проблема с переносом

marahin1992 committed:
created modal function

Eugetis committed:
Merge branch 'develop' into feat/modal-functions

Eugetis committed:
Merge pull request #63 from Eugetis/feat/modal-functions

created modal function

AlexxxDevv committed:
появляющийся при скролле хедер

Commit:
feat: script drag-n-drop

Eugetis committed:
Merge branch 'develop' into header-with-js-2

Eugetis committed:
Merge pull request #64 from Eugetis/header-with-js-2

появляющийся при скролле хедер

AlexxxDevv committed:
проверка страницы для хедера

AlexxxDevv committed:
Merge branch 'header-with-js-2' of github.com:Eugetis/hedonism-fest into header-with-js-2

Commit:
Merge branch 'develop' into feat/participants

Eugetis committed:
fix: small layout fixes

Eugetis committed:
Merge pull request #65 from Eugetis/header-with-js-2

Header with js 2

Eugetis committed:
Merge pull request #66 from Eugetis/fix/layout

fix: small layout fixes

marahin1992 committed:
floating-btn

Eugetis committed:
Merge pull request #67 from Eugetis/feat/scroll-btn

floating-btn

AlexxxDevv committed:
header fixes

AlexxxDevv committed:
header fixes

Eugetis committed:
Merge pull request #68 from Eugetis/header-with-js-3

Header with js 3

Eugetis committed:
ui-kit: info-message

Eugetis committed:
Merge pull request #69 from Eugetis/feat/info-message

ui-kit: info message

Commit:
Добавил открытие/закрытие модалки

Eugetis committed:
Merge pull request #70 from Eugetis/modal/card-template

Добавил открытие/закрытие модалки

dmitry-razumov committed:
feat: реализована фильтрация по таг кнопкам и прокинут вариант рендера карточек по ее результатам

dmitry-razumov committed:
merge: dev to catalog/filtration

Eugetis committed:
Merge pull request #71 from Eugetis/catalog/filtration

feat: catalog filtration

Commit:
Сделал копирование ссылки

Eugetis committed:
Merge pull request #72 from Eugetis/modal/card-template

Сделал копирование ссылки

Eugetis committed:
feat: modal with favs and extra addresses

Eugetis committed:
Merge pull request #73 from Eugetis/feat/modal-address

feat: modal with favourites and extra address

Commit:
feat/modal-card-address

Eugetis committed:
Merge pull request #74 from Eugetis/modal/card-template

feat/modal-card-address

Commit:
fix: update field styles/ start validation

AlexxxDevv committed:
выезжающий слайдер

Eugetis committed:
Merge pull request #75 from Eugetis/feat/participants

feat: update fields style, basic validation

Commit:
feat/favorite-cards

Eugetis committed:
feat: start modal forms layout

Eugetis committed:
Merge pull request #77 from Eugetis/modal/card-template

feat/favorite-cards

Eugetis committed:
Merge pull request #78 from Eugetis/feat/modal-forms

feat: modal forms - start layout

Commit:
feat/favorite-card-fix

AlexxxDevv committed:
bug fixes

Eugetis committed:
Merge pull request #79 from Eugetis/modal/card-template

feat/favorite-card-fix

Eugetis committed:
fix: some small fixes

AlexxxDevv committed:
fixes

AlexxxDevv committed:
fixes

AlexxxDevv committed:
fixes

Eugetis committed:
Merge pull request #76 from Eugetis/header-with-js-4

выезжающий слайдер

marahin1992 committed:
fixed modal close btn

Eugetis committed:
Merge pull request #80 from Eugetis/fix/close-btn

fixed modal close btn

dmitry-razumov committed:
feat: события иконками на карте
refact: выделил код map и tag-filter в отдельныевыделил
todo: baloon<>modal, fix days filter

dmitry-razumov committed:
Merge branch 'develop' into catalog/map-events

dmitry-razumov committed:
merge with dev, fix: size clusterer

Eugetis committed:
refact: forms, no js

dmitry-razumov committed:
feat: кастомный балун из модалки,
повешен слушатель на кнопку, вызывает хэндлер,
там надо вызвать функцию другой модалки вместе с передачей id
map.js:60:(handleButtonCustomBaloon)
todo: fix кнопок фильтров, пустую страницу

Eugetis committed:
Merge pull request #81 from Eugetis/catalog/map-events

feat: catalog-map events

Eugetis committed:
Merge pull request #82 from Eugetis/catalog/map-ballon-modal

feat: catalog-map-ballon-modal

Eugetis committed:
feat: payment form layout

Eugetis committed:
fix: forms layout

Eugetis committed:
Merge branch 'develop' into feat/modal-forms

Eugetis committed:
Merge pull request #83 from Eugetis/feat/modal-forms

feat&refact: forms

dmitry-razumov committed:
feat: кнопка все автоматом вкл когда все выкл, refact: слушатель кнопок и набор активных тагов

Eugetis committed:
Merge pull request #84 from Eugetis/feat/tag-filter-button-all

feat: tag filter button all

dmitry-razumov committed:
fix: ширина мобайл захардкожена для стиля балуна

Eugetis committed:
Merge pull request #85 from Eugetis/catalog/map-balloon-mobile-fix

fix: catalog-map balloon mobile fix

Eugetis committed:
fix: preview layout

Eugetis committed:
Merge pull request #86 from Eugetis/fix/all

fix: preview layout

Commit:
...

Commit:
feat: modal opened by click more in the map card

Eugetis committed:
Merge pull request #87 from Eugetis/modal/card-template

Modal/card template

Commit:
feat: modal likes fixed

Commit:
feat: added price to payment modal

Commit:
feat: payment interactive added

Eugetis committed:
Merge pull request #88 from Eugetis/modal/card-template

Modal/card template

Commit:
fix: remove modal

Commit:
fix: refactoring

Eugetis committed:
Merge pull request #89 from Eugetis/modal/card-template

fix: remove modal

dmitry-razumov committed:
fix: remove modal, css class для поля тип события в верстке модалки балуна

dmitry-razumov committed:
Merge branch 'develop' into modal/card-template

Eugetis committed:
Merge pull request #90 from Eugetis/modal/card-template

fix: modal card template

Commit:
fix: api url

Commit:
fix: api

Eugetis committed:
Merge pull request #91 from Eugetis/modal/card-template

Modal/card template

Commit:
fix: modal mobile preview

Eugetis committed:
Merge pull request #92 from Eugetis/modal/card-template

fix: modal mobile preview

Commit:
feat: added cards to 404, thanks pages

Eugetis committed:
Merge pull request #93 from Eugetis/modal/card-template

feat: added cards to 404, thanks pages

Eugetis committed:
feat: animations first step

Eugetis committed:
Merge pull request #94 from Eugetis/feat/animations

feat: animations first step

Commit:
fix: card click dataset

Eugetis committed:
Merge pull request #95 from Eugetis/modal/card-template

fix: card click dataset

Commit:
fix: modal remove

Commit:
fix: currency in payment modal

Eugetis committed:
Merge pull request #96 from Eugetis/modal/card-template

fix: modal remove

Commit:
fix: payment input

Eugetis committed:
feat: animations done

Eugetis committed:
Merge pull request #97 from Eugetis/modal/card-template

fix: payment input

Eugetis committed:
Merge pull request #98 from Eugetis/feat/animations

feat: animations done

Commit:
fix: mobile filter modal

Eugetis committed:
Merge pull request #99 from Eugetis/modal/card-template

fix: mobile filter modal

Commit:
feat: added redirect to catalog by card click, from index, 404, 2 thanks pages

Eugetis committed:
Merge pull request #100 from Eugetis/modal/card-template

feat: added redirect

Commit:
fix: card redirect

Commit:
feat: modal favorite open

Eugetis committed:
Merge pull request #101 from Eugetis/modal/card-template

Modal/card template

dmitry-razumov committed:
feat: tab-switch for modal, refact: tab-switch code one for both

Eugetis committed:
Merge pull request #102 from Eugetis/feat/tab-switch-and-formodal

feat: tab-switch for modal, refact: tab-switch code one for both

Eugetis committed:
refact: geo desktop

dmitry-razumov committed:
feat: map in favor, но после закрытия модалки вырезается нода с картой из секции на основной странице

Commit:
added map cloneNode

dmitry-razumov committed:
feat: map in modal favorites

Eugetis committed:
Merge pull request #103 from Eugetis/modal/favorites-map

feat: modal favorites map

Eugetis committed:
fix: header (partly)

Eugetis committed:
Merge pull request #104 from Eugetis/fix/header

fix: header (partly)

Eugetis committed:
fix: small

dmitry-razumov committed:
broken, in process

Commit:
fix: fixed donation modal

Eugetis committed:
Merge pull request #105 from Eugetis/modal/fix

fix: fixed donation modal

dmitry-razumov committed:
Merge branch 'develop' into catalog/no-events

dmitry-razumov committed:
feat: карты везде
fix: поломку с закрытиями\открытиями модалок\карты переключений
feat: страница когда нет лайков

Eugetis committed:
Merge pull request #106 from Eugetis/feat/maps-everywhere-fix-modal-nolikepage

feat: Feat maps everywhere fix modal nolikepage

Eugetis committed:
fix/misc

Eugetis committed:
Merge pull request #107 from Eugetis/fix/misc

fix: misc

Eugetis committed:
add link to brief

Commit:
fix: fixed donation modal

Commit:
fix: dev update

sazonov-dev committed:
Merge pull request #109 from Eugetis/modal/fix

Modal/fix

Commit:
fix: fixed favorite redirect

sazonov-dev committed:
Merge pull request #110 from Eugetis/modal/fix

fix: fixed favorite redirect

coolswood commented:
Работа отклоняется без потери итерации, так как бриф не доступен по приведенной ссылке.

Commit:
fix: fixed cards

Eugetis committed:
Merge pull request #111 from Eugetis/modal/fix

fix: fixed cards

Commit:
fix: modal favorite fixed

Eugetis committed:
Merge pull request #112 from Eugetis/modal/fix

fix: modal favorite fixed

coolswood commented:
## Критерии

1. Соответствие макету:
    - [ ] <!--12.0--> 1.1 Все элементы из макета сверстаны
`отсутсвуют одиночные флажки на карте. При приближении множественный флажок должен пропадать и показывается одиночное мероприяятие. Сейчас только множественные`
<img width="415" alt="Screenshot 2023-09-06 at 14 25 12" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/8d22f10d-63c8-413c-949e-f90c34129860">

`Отсуствует состояние кнопки в карточке: Крестик, красное сердце (есть только черное) Скрин из макета`
<img width="246" alt="Screenshot 2023-09-06 at 14 14 33" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/a6c85e18-0f6d-4248-92d2-a50c8c0a9535">

`Отсутствует иконка крестик на фильтре "Хочу пойти"`
<img width="311" alt="Screenshot 2023-09-06 at 14 24 13" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/39e0f2cc-c6b7-4dba-afe9-e29aac9f8e81">

`отсутствует иконка "Галочка" при выборе города в мобильной версии`
<img width="301" alt="Screenshot 2023-09-06 at 16 46 41" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/72168b58-922a-4c4e-bcd3-d8396801cd80">

`После отправки заявки не открывается страница с успешным отправлением заякви`
`Все поля должны работать. Не увидел в брифе что их надо дизейблить`
<img width="455" alt="Screenshot 2023-09-06 at 17 03 19" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/23593773-de23-410f-90f1-ca3e564e1e58">


    - [x] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету
    - [ ] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета
`Пункт брифта: 1.4.1. если нет сохраненных, иконка не залитая, при нажатии открывается - “каталог, список, пусто”. Отсутствует состояние пустого списка. На экране должна отображаться картинка (https://www.figma.com/file/c3aY4aPf79ICMTqoC4AKLo/%D0%9D%D0%BE%D1%87%D0%BB%D0%B5%D0%B6%D0%BA%D0%B0-(Polished)?type=design&node-id=1990-9337&mode=design&t=OfMNlXjOFj2dILeJ-4)`
<img width="1307" alt="Screenshot 2023-09-06 at 14 11 39" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/40b9d4af-9597-40c7-b291-62f66cec3ae1">

`На главной странице в слайдере в мобильной версии должны быть карточки со скругленными углами, так как в макете прорисованы именно такие (1 скрин - макет, 2 скрин - ваша работа)`
<img width="293" alt="Screenshot 2023-09-06 at 16 31 06" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/dabb1cde-4d38-4917-8686-3187a93fae13">
<img width="368" alt="Screenshot 2023-09-06 at 16 31 14" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/8fe568a6-1ec8-4f89-89fb-39fafa5349f9">

`Кнопки в мобильной версии должны располагаться в нижней части экрана`
<img width="323" alt="Screenshot 2023-09-06 at 16 30 06" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/05255b63-9c51-4209-a5c4-9ae7c2fcd704">

`Отсутствуют отступы между блоками`
<img width="514" alt="Screenshot 2023-09-06 at 14 14 08" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/7af5935d-007d-40ec-8619-7c63faace6fe">

`Контент прилипает к хедеру`
<img width="735" alt="Screenshot 2023-09-06 at 14 16 02" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/2aa438c8-3193-4868-a1bd-fb235fab1698">


    - [ ] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser
`Селект выбора города на стр заявки не соответствует макету`
<img width="628" alt="Screenshot 2023-09-06 at 16 20 23" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/e05cc5f9-dbba-4464-ab2e-6acd2bdc3891">

`Выбранный город не должен выделяться фоном как при наведении`
<img width="382" alt="Screenshot 2023-09-06 at 14 09 26" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/b354404c-d0b7-4667-8b4e-e4bdd6642bff">



2. Адаптив:
    - [x] <!--4.0--> 2.1 На всех размерах экрафна 320px и более отсутствует горизонтальный скролл
    - [x] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету
    - [x] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами
    - [x] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один
    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более 
    - [ ] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
`В мобильной версии картинки не должны обрезаться и быть вытянутыми`
<img width="329" alt="Screenshot 2023-09-06 at 16 55 18" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/c6f8e20b-2b0f-42ed-a43b-e03d1588107f">

    - [x] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются
3. Верстка:
    - [ ] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)
`Необходимо проверить валидацию.  Вы можете проверить ее здесь (https://validator.w3.org/)`

    - [x] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо
    - [x] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>`
    - [x] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки 
    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [x] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы
    - [x] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`.
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [x] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении
    - [ ] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность

    - [x] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта
5. Применение БЭМ:
    - [ ] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют
`Есть ошибки в нейминге, Вы можете проверить их здесь (https://nglazov.github.io/bem-validator-page/)`

    - [x] <!--2.0--> 5.2 Не используются селекторы по тегу
    - [x] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [x] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)
    - [x] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства
    - [x] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested
- [ ] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект)
`Ни од`
7. Структура проекта и оформление кода
    - [x] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта
    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов

## Доп. комментарии:
`При неоднократном нажатии на кнопку, добавляется большое кол-во элементов`
<img width="368" alt="Screenshot 2023-09-06 at 14 26 48" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/f698bfe0-00dc-4965-b0eb-d54c8d434424">

`Переодически появляется ошибка`
<img width="1140" alt="Screenshot 2023-09-06 at 14 12 21" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/0654e743-e7af-4d47-b062-4455087ca599">

`Этого нет в макете, но странно что логотипы не скролятся в мобильной версии`
<img width="338" alt="Screenshot 2023-09-06 at 16 45 15" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/1e2f3dfc-7f89-4fcf-9e8a-96af01987185">

`Хедер должен всегда висеть на странице`
<img width="1424" alt="Screenshot 2023-09-06 at 14 11 23" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/18b1ae20-be02-4433-bf3c-baf37062e1bb">



## Количество баллов: 57

##

marahin1992 committed:
fixed gallery and added scroll in team section

Eugetis committed:
Merge pull request #113 from Eugetis/fix/gallery_mobile

fixed gallery and added scroll in team section

Commit:
fix: api

marahin1992 committed:
some validation

Eugetis committed:
Merge pull request #114 from Eugetis/modal/fix

fix: api

Eugetis committed:
Merge pull request #115 from Eugetis/fix/validation

some validation

dmitry-razumov committed:
refact: filtration, broken, in process

dmitry-razumov committed:
fix: check favorites, create map

Eugetis committed:
Merge pull request #116 from Eugetis/fix/check-fav-and-ymaps

fix: check favorites, create map

dmitry-razumov committed:
merge: dev & fix/check-fav-ymaps

Eugetis committed:
fix: delete additional header on the main page

Eugetis committed:
Merge pull request #117 from Eugetis/fix/header-layout-and-js

fix: delete additional header on the main page

Eugetis committed:
fix: misc

Eugetis committed:
Merge pull request #118 from Eugetis/fix/layout-misc

fix: misc

Eugetis committed:
fix: dropdown close

Eugetis committed:
Merge pull request #119 from Eugetis/fix/layout-misc

fix: dropdown close

dmitry-razumov committed:
refact: фильтрация, в модальных окнах результат на картах также
отфильтрован
feat: в хидере хочу пойти отрабатывает активность лайк
refact: корректная логика хочу пойти, и кнопка, и в хидере
добавлена активность кнопки на странице ой

dmitry-razumov committed:
Merge branch 'develop' into refact/filtration

Eugetis committed:
Merge pull request #120 from Eugetis/refact/filtration

refact: filtration map

dmitry-razumov committed:
fix: filters when all

Eugetis committed:
fix: for final review

Eugetis committed:
Merge pull request #121 from Eugetis/refact/filtration

fix: filters when all

Eugetis committed:
readme and w3c validation

coolswood commented:
## Критерии

1. Соответствие макету:
    - [x] <!--12.0--> 1.1 Все элементы из макета сверстаны
    - [x] <!--5.0--> 1.2 Шрифты, размер, цвет и начертания текста соответствуют макету
    - [x] <!--5.0--> 1.3 Ширина элементов и отступы отличаются от макета не более, чем на 30px на десктопной версии и на 10px на остальных размерах экрана, для которых подготовлен макет. Отличия в размерах не должны визуально противоречить макету и выбиваться из заложенной в макет сетки, в этом случае требуется более строгое соответствие размерам макета
    - [ ] <!--3.0--> 1.4 Все блоки соответствуют макету в последних версиях Firefox, Google Chrome или Yandex Browser
`Селект выбора города на стр заявки не соответствует макету. Иконка селекта съехала`
<img width="470" alt="Screenshot 2023-09-11 at 22 08 14" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/d137126e-20fc-4ba0-afd3-8c0b3760cc94">
<img width="628" alt="Screenshot 2023-09-06 at 16 20 23" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/e05cc5f9-dbba-4464-ab2e-6acd2bdc3891">

2. Адаптив:
    - [x] <!--4.0--> 2.1 На всех размерах экрафна 320px и более отсутствует горизонтальный скролл
    - [x] <!--1.0--> 2.2 Установленная максимальная ширина контента соответствует макету
    - [x] <!--4.0--> 2.3 Вёрстка не ломается между брейкпоинтами
    - [x] <!--1.0--> 2.4 Брейкпоинты сгруппированы. Если два брейкпоинта имеют небольшую пиксельную разницу в медиа-правиле, они объединены в один
    - [x] <!--4.0--> 2.5 Текст не выпадает из блоков на всех размерах экрана 320px и более 
    - [x] <!--3.0--> 2.6 Изображения не искажаются и выглядят как в макете на всех размерах экрана
    - [x] <!--1.0--> 2.7 Одинаковые свойства в разных медиа-правилах не дублируются
3. Верстка:
    - [ ] <!--4.0--> 3.1 Нет ошибок валидации [https://validator.w3.org/nu/](https://validator.w3.org/nu/) (Erorrs)
`Необходимо проверить валидацию.  Вы можете проверить ее здесь (https://validator.w3.org/)`

    - [x] <!--3.0--> 3.2 Используются `<main>`, `<section>`, `<header>`, `<footer>`, `<ul>`, `<nav>`, `<h1-h6>` и `<p>` там, где это необходимо
    - [x] <!--2.0--> 3.3 Не используются элементы `<b>`, `<br>`, `<i>`
    - [x] <!--1.0--> 3.4 Отсутствуют лишние `div` обёртки 
    - [x] <!--3.0--> 3.5 Контентные картинки вставлены через `<img>` и в них присутствует `alt` с описанием, соответствующим языку страницы
    - [x] <!--2.0--> 3.6 На странице есть заголовок первого уровня
    - [x] <!--2.0--> 3.7 Заголовки соответствуют иерархии страницы
    - [x] <!--1.0--> 3.8 Корректно задан `viewport`, прописаны `title` и `lang`.
4.  CSS:
    - [x] <!--2.0--> 4.1 Не используется `!important`
    - [x] <!--2.0--> 4.2 Абсолютное позиционирование не используется без непосредственной необходимости
    - [x] <!--4.0--> 4.3 Использование `height` и `max-height` не приводит к выпадению текста или блоков при переполнении
    - [x] <!--2.0--> 4.4 Любой интерактивный элемент имеет состояние наведения, состояние наведения берется из макета или используется прозрачность
    - [x] <!--1.0--> 4.5 Для каждого шрифта указаны альтернативные варианты из системных шрифтов и тип шрифта
5. Применение БЭМ:
    - [ ] <!--4.0--> 5.1 Нет нарушения методологии БЭМ: нет элементов без родительского блока, нет элементов элементов, модификаторы не используются без указания того, что они модифицируют
`Есть ошибки в нейминге, Вы можете проверить их здесь (https://nglazov.github.io/bem-validator-page/)`

    - [x] <!--2.0--> 5.2 Не используются селекторы по тегу
    - [x] <!--3.0--> 5.3 Для одинаковых блоков в макете переиспользуется стили одного БЭМ-блока
    - [x] <!--1.0--> 5.4 Числа в названиях классов не используются, за исключением классов для элементов, указанных в руководстве по стилю (Style Guide)
    - [x] <!--1.0--> 5.5 Модификатор не отражает позицию элемента в разметке (`paragraph_first`)
    - [x] <!--1.0--> 5.6 Модификаторы не содержат дубли стилей модифицируемого элемента или блока, только изменяющиеся свойства
    - [x] <!--2.0--> 5.7 Файловая структура построена по БЭМ nested
- [ ] <!--10.0--> 6. Функциональность на JS работает корректно (слайдеры, бургерное меню или другие требования согласно заданию на проект)
`Некоторые формы некорректно валидируются. В номере телефона необходимо ограничить символы и сделать удобное отображение написания телефона, стандарт +7 (000) 00 00`
<img width="578" alt="Screenshot 2023-09-11 at 22 07 54" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/c3f8bf08-a818-4704-98db-a6891bea5ccf">

`Некоторые формы некорректно валидируются. Часы работы должны иметь определенный формат` 
<img width="488" alt="Screenshot 2023-09-11 at 22 09 02" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/ee9a6e81-c681-4ce4-80f8-fc32f92deac6">

7. Структура проекта и оформление кода
    - [x] <!--2.0--> 7.1 Соблюдена единообразная файловая структура проекта, например, изображения в папке `images`, шрифты в папке `fonts`, стили в папке `blocks`, скрипты в папке `scripts`, а не разбросаны в разных папках и не хранятся в корне проекта
    - [x] <!--1.0--> 7.2 Код в проекте отформатирован единообразно
    - [x] <!--1.0--> 7.3 Нет нарушений иерархии отступов в разметке, стилях и файлах скриптов

## Доп. комментарии:
`Переодически появляется ошибка`
<img width="1140" alt="Screenshot 2023-09-06 at 14 12 21" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/0654e743-e7af-4d47-b062-4455087ca599">

`Ошибка при клике на "хочу пойти"`
<img width="1010" alt="Screenshot 2023-09-11 at 22 04 13" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/1aa1da56-2b4c-4823-9520-0d53674730b3">

 `При выборе города в мобильном меню, город (class="geo__button geo__button_type_mobile button") не меняется.`
`Лучше не применять разные написания городов (разные языки)`
<img width="301" alt="Screenshot 2023-09-11 at 22 12 04" src="https://github.com/Eugetis/hedonism-fest/assets/25247802/e5707c7c-f79f-45ce-a602-72cb6b0cd23e">

## Количество баллов: 79

##
